### PR TITLE
Add .ndim and .T to Tensor (OpenMined#4617)

### LINKED ISSUE #4617: Transpose method `torch.Tensor.T` does not work
## Description
When performing the transpose operation `x.T` (where x is a tensor in a worker) returns an empty tensor instead of a pointer. Instead, it is necessary to use `.transform(1,0)`

## How to Reproduce
```python
import torch
import syft as sy

hook = sy.TorchHook(torch)
bob = sy.VirtualWorker(hook, id='bob')

x = torch.rand(250, 84).send(bob)  # Synthetic tensor
x.T

# Output
# tensor([])
```

## Expected Behavior
Should return a pointer containing the transposed tensor.

## System Information
- The official released Docker container
- Same for pip package:
  - Ubuntu 18.04.5 LTS (Bionic Beaver)
  - Python 3.6.9

### PR DESCRIPTION
**Pull request by xutongye**
## Description
Fixes #4617 
Add .ndim and .T functionality to FixPrecisionTensor and PointerTensor.

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- test units

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: test/torch/tensors/test_precision.py [hunk 1]
```diff
@@ -693,6 +693,16 @@ def test_dtype():
...
 
 
+def test_ndim():
+    x = torch.rand(2, 3, 4).fix_prec()
+    assert x.ndim == 3
+
+
+def test_T():
+    x = torch.rand(2, 3, 4).fix_prec()
+    assert x.T.shape == torch.Size([4, 3, 2])
```
Comment (gmuraru) [inline test/torch/tensors/test_precision.py:703]: Test the value for transpose

File: test/generic/pointers/test_pointer_tensor.py [hunk 2]
```diff
@@ -379,6 +379,21 @@ def test_get_remote_shape(workers):
...
+    x = th.rand(2, 3, 4).send(bob)
+    assert x.ndim == 3
+
+
+def test_remote_T(workers):
+    """Test pointer.T functionality"""
+    bob = workers["bob"]
+    x = th.rand(2, 3, 4).send(bob)
+    y = x.T
+    assert y.shape == torch.Size([4, 3, 2])
```
Comment (gmuraru) [inline test/generic/pointers/test_pointer_tensor.py:394]: Here the same, test "y" not the shape of y

File: test/torch/tensors/test_precision.py (hunk 1 above)
Comment (xutongye) [reply test/torch/tensors/test_precision.py]: done in a new commit, by testing `x.fix_prec().T==x.T.fix_prec()`

File: test/generic/pointers/test_pointer_tensor.py (hunk 2 above)
Comment (xutongye) [reply test/generic/pointers/test_pointer_tensor.py]: doen in a new commit, by testing `bob_xT.get()==x.T` and `bob_x.get()==x`

**Comment by xutongye** [comment]
The last commit is I rebase the previous one to the lastest master branch.

**Review by gmuraru — APPROVED** [review]
LGTM!
